### ISSUE TITLE
Formatting doesn't work for me

### ISSUE THREAD

**Issue by npearson72** [pantharshit00/coc-prisma#22]
I've followed the instructions here:

https://github.com/pantharshit00/coc-prisma/blob/fc64aaa7fb0f0cdf51a84eeadee53e99fe1d3b90/README.md#formatting

However, when using the command `:Format` I get the following: `E492: Not an editor command: Format`

Also, here is what I have in my coc-settings:

```
...
  "coc.preferences.formatOnSaveFiletypes": [
    "css",
    "html",
    "javascript",
    "javascriptreact",
    "json",
    "jsonc",
    "markdown",
    "prisma",
    "ruby",
    "scss",
    "typescript",
    "typescriptreact",
    "yaml"
  ],
...
```

But it does not work for prisma. All other filetypes do format properly.

Any ideas?

**Comment by lukevers** [comment]
Yeah I'm not sure either..it almost looks like that's defined from the prettier plugin and not COC directly? I tried the prettier plugin, but it doesn't support prisma lol

**Comment by SOLID-arity** [comment]
Thanks @lukevers that works great. Any ideas how to get it to work with syntax highlighting? I normally use coc-prettier. The gif in the readme has highlighting, so someone got it to work..

**Comment by lukevers** [comment]
@SOLID-arity Yeah I'm using https://github.com/pantharshit00/vim-prisma

Here's my vimrc if it helps https://github.com/lukevers/vimrc/blob/main/vimrc#L17
